Dockerfile-linux.template

```diff
@@ -10,16 +10,6 @@ FROM alpine:{{ alpine_version }}
 
 RUN apk add --no-cache ca-certificates
 
```

Comment:
Off-by-one; this empty line should go too :sweat_smile: 

```suggestion
```

(Again, happy to take care of this nit for you if you'd prefer!)

Reply:
:cry: